### DISCUSSION TITLE
Is there a guide or template on multi-step forms?

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#5408] [Help] [answered]
Trying to clean up and simplify a complex screen with to many options picker and fields, I am looking for a guide or template on multi-step forms. I apologize if my google-fu is not strong enough to find an example myself.

**Comment by melohagan** [comment, marked as answer]
Hi @gerhardmpl 

Thank you for highlighting this! I have added a new docs page to demonstrate multi-step forms: https://docs.budibase.com/docs/form-steps

If there's anything else you need clarification on, don't hesitate to ask!

**Reply by gerhardmpl** [reply]
Thank you, that helps me get started!
